components/setup.sh

```diff
@@ -22,7 +22,18 @@ function setup() {
     bundle exec fastlane match appstore --readonly $verboseArg
   fi
 
+  msg 'Installing Git hooks'
+  symlinkGitHooks
+
   comp_deinit
 
   carthage_bootstrap
 }
+
+function symlinkGitHooks() {
+  hooksDir="$project_dir"/.git/hooks
+  mkdir -p "$hooksDir"
+  ln -s ../../bin/git-hooks/post-checkout "$hooksDir"
+  ln -s ../../bin/git-hooks/post-merge "$hooksDir"
```

Comment:
@aexmachina i just re-used the same technique that was used from the Carthage.tgz implementation, but you're right - it does error out if the file exists as i found out when i added the new post-merge file. this is why i separated them out to 2 lines, since in a normal bash script the 2nd would run regardless of the first, but i just remembered that with `set -o errexit` it will halt the entire script..  perhaps i need to do `ln -fs` ? then i can put the {file1,file2} back in